osc-domain/src/main/java/org/osc/core/broker/model/entities/appliance/VirtualSystem.java

```diff
@@ -233,6 +233,18 @@ public Boolean getMarkedForDeletion() {
         return super.getMarkedForDeletion() || this.distributedAppliance.getMarkedForDeletion();
     }
 
+    public String getRedirectionTargetId() {
```

Comment:
this is not accurate. We cannot just get the first redirection target id from the VS. this has to be at the DS level only.

Also rename redirection id to ssomething more generic. work with karim and myself to come up with suggestions.